format.md

```diff
@@ -124,6 +133,29 @@ is a list of xy vertices. The child name of the outer list should be "polygons";
 the child name of the middle list should be "rings"; the child name of the
 inner list should be "vertices".
 
+### Serialized encodings
+
+**Well-known binary (WKB)**: `Binary` or `LargeBinary` or `FixedSizeBinary`
```

Comment:
Do you know if FixedSizeBinary would be used in practice? 
I can imagine that for points it could be a bit useful, but on the other hand, it doesn't give that much memory reduction (given that each value already has a considerable length). In contrast to fixed size lists (for interleaved), fixed size binary _is_ supported by Parquet, though.